hs_access_control/models.py

```diff
@@ -2298,7 +2298,8 @@ def view_users(self):
                                       u2urp__privilege__lte=PrivilegeCodes.VIEW) |
                                     Q(u2ugp__group__g2grp__resource=self.resource,
                                       u2ugp__group__g2grp__privilege__lte=PrivilegeCodes.VIEW)))\
-                           .exclude(u2ugp__group__gaccess__active=False).distinct()
+                           .exclude(Q(u2ugp__group__gaccess__active=False,
```

Comment:
Just move u2ugp__group__gaccess__active=True in the upper phrase.
Not sure why it's like this, but it serves no purpose.  Expression will be
```
return User.objects.filter(Q(is_active=True) &
                            (Q(u2urp__resource=self.resource,
                               u2urp__privilege__lte=PrivilegeCodes.CHANGE) |
                             Q(u2ugp__group__gaccess__active=True, 
                               u2ugp__group__g2grp__resource=self.resource,
                               u2ugp__group__g2grp__privilege__lte=PrivilegeCodes.CHANGE))
                       .distinct()
```